third_party/com_github_grpc_ecosystem_grpc_gateway/protoc-gen-grpc-gateway/BUILD.in

```diff
@@ -0,0 +1,41 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
```

Comment:
This repo already has build files. Why do they need to be replaced? Can they be modified there instead?

Reply:
yes that is the best strategy. see comments below.